### PR TITLE
Fix lozenges

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- The Lozenges aren't written in the same way as other components on the toolkit.
- They don't include sizes, or utilise the colours in UI-Toolkit, they rely on bootstrap being introduced to be styled
- This PR addressed that issue and adds in Toolkit styling.

#### What tests does this PR have?
- I have added in more test to check size classes are being added to the element.

#### How can this be tested?
- Run `grunt docs` is the best place to start
- Once the docs have opened click on `lozenges` in the navigation
- You should see a full set of beautiful lozenges, you can edit to code by clicking on the `code` tab ont eh doc (it looks like you can't edit it but try using backspace.

#### Screenshots / Screencast
screen shot 2015-08-11 at 14 55 34

#### What gif best describes how you feel about this work?
adventure time

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

### CODE REVIEW

**Comment by robhuzzey** [comment]
Taking SE/DEV

File: src/components/lozenge/code/templates/LozengeComponentTemplate.jsx [hunk 1]
```diff
@@ -1,21 +1,15 @@
-var React = require('react/addons');
-var classnames = require('classnames');
+var React = require('react');
+var classNames = require('classnames');
+var getComponentClasses = require('../../../../utils/getComponentClasses');
```
Comment (robhuzzey) [inline src/components/lozenge/code/templates/LozengeComponentTemplate.jsx]: I still hate that we have to do `../../../../../../......` all the time :(

**Comment by robhuzzey** [comment]
Looks good to me, one comment I do have however is to why we have `src/components/lozenge/code/templates/LozengeComponentTemplate.jsx` called into `src/components/lozenge/code/views/LozengeComponentView.jsx` * we couldn't just use the one file (seeing as we call the function with switched context anyway).

I think the original though around this was that we may re-use the JSX templates but I haven't seen a use case for it yet?

Not a blocker, just something I wanted to air :)

**Comment by jodiedoubleday** [comment]
I completely agree @robhuzzey we will be looking to update all out components soon.

File: docs/src/Components.jsx [hunk 2]
```diff
@@ -134,7 +134,7 @@ var Components = React.createClass({
           <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/Lozenge.jsx', 'utf8')} />
           <h4>Attributes</h4>
           <ul>
-            <li><code>purpose</code> String - default, primary, secondary, success, info, warning, danger</li>
+            <li><code>purpose</code> String - default, primary, secondary, success, info, warning, danger, block</li>
```
Comment (andrew-c) [inline docs/src/Components.jsx]: Need to document the `size` property.

**Comment by andrew-c** [comment]
:+1:

**Pull request merged**